sdk/core/core-asynciterator-polyfill/ThirdPartyNotices.txt

```diff
@@ -0,0 +1,35 @@
+Third Party Notices for ms-rest-js
```

Comment:
`ms-rest-js` -> `core-asynciterator-polyfill`

Reply:
Is it third-party though?